docker-compose.yml

```diff
@@ -49,7 +49,7 @@ services:
       - ~/.aws/:/root/.aws:ro
       - ./logs:/app/logs
     deploy:
-      replicas: 2
+      replicas: 5
```

Comment:
this may look exaggerated, but with only 2 it was taking forever to fetch IPFS stuff because of the new timeout of 5 seconds. No problem when indexing data that is available on IPFS, but if the data is not found we have the 5 seconds * 2 nodes (ipfs and aquamarine) * 5 retries